hbase-shell/src/main/ruby/hbase/admin.rb

```diff
@@ -591,6 +591,10 @@ def unassign(region_name, force = nil)
     #----------------------------------------------------------------------------------------------
     # Move a region
     def move(encoded_region_name, server = nil)
+      if encoded_region_name == 'hbase:meta'
```

Comment:
I think it would be much better if the Admin API supported this feature directly. That way, it works the same way, everywhere. Is there a strong reason to implement it only in the shell?